### PR TITLE
Update base_url to api.vipps.no and add more notes to the postman guide

### PR DESCRIPTION
**Pull request by rebeksburnett**

### CODE REVIEW

File: vipps-partner-postman.md [hunk 1]
```diff
@@ -15,22 +15,35 @@ We also have a short [getting started guide to Postman](https://github.com/vipps
...
+  The Partner API only works in the production environment, so `base_url` is set to `api.vipps.no`.
+
+### Step 4: Run Tests
+
+Be aware that these are running on the production server.
+
+1. Send request `Get Access Token`. This provides you with access to the API.
+2. Send request `Get sale unit details based on MSN`. This returns a JSON structure with the details, including the org number. If necessary, update `orgno` in the environment.
+3. Send `Get merchant by organization number` for details about the transaction.
+4. Open `Order products on behalf of merchants` to see an example of ordering products. Since this is running on the production server, you might not want to run it.
```
Comment (niklud) [inline vipps-partner-postman.md:46]: These are independent of each other, so numbering them as steps is kinda wrong :thinking:

File: vipps-partner-postman.md [hunk 2]
```diff
@@ -15,22 +15,35 @@ We also have a short [getting started guide to Postman](https://github.com/vipps
...
-1. Click the "eye" icon in the top right corner.
-2. In the dropdown window, click `Edit` in the top right corner.
+1. Click the down arrow, next to the "eye" icon in the top-right corner, and select the environment you have imported.
+2. Click the "eye" icon and, in the dropdown window, click `Edit` in the top-right corner.
 3. Fill in the `Current Value` for the following fields to get started.
    - `client-id`
    - `client-secret`
+   - `merchantSerialNumber`
    - `Ocp-Apim-Subscription-Key`
+   - `orgno`
```
Comment (niklud) [inline vipps-partner-postman.md:35]: I'm not sure if it needs mention, but orgno is only needed for `Get merchant by organization number`and merchantSerialNumber is only needed by `Get sales unit details based on MSN`

**Review by niklud — CHANGES_REQUESTED** [review]
Maybe add some clarifications, or do we want to make them always follow the same flow when testing with Postman?

File: vipps-partner-postman.md (hunk 1 above)
Comment (rebeksburnett) [reply vipps-partner-postman.md:46]: I see what you mean. I like having them in a numbered order - kind of like a lesson. I will mention that they can run them independently...
I think this is the best order to run them in.

**Review by niklud — APPROVED** [review]
LGTM

**Pull request merged**